cylc/flow/scheduler_cli.py

```diff
@@ -208,6 +208,13 @@ def get_option_parser(is_restart):
         ),
         metavar="HOST", action="store", dest="host")
 
+    parser.add_option(
+        "--format",
+        help="The format of the output: 'plain'=human readable, 'json'",
```

Comment:
Would it be useful to tell users that this only works if `--no-detach` is not used? Submitted a workflow twice before realizing it myself :grin:

Reply:
Should probably make the options orthogonal.

Reply:
Done.